js/main.js

```diff
@@ -22,4 +35,59 @@ const getRandomFractional = (min, max, digits = 5) => {
   return Number(result);
 };
 
-getRandomFractional(1, 4);
+// Функция для получения случайного элемента из массива
+
+const getRandomArrayElement = (element) => element[getRandomNumber(0, element.length - 1)];
+
+// Функция для получения нескольких случайных элементов из массива
+
+const getRandomArrayElements = (elements) => {
+  const getShuffledArray = (elements).sort(()=>Math.random()-0.5); // Перемешивание массива
```

Comment:
Не забывай операторы отделять пробелами
```suggestion
  const getShuffledArray = (elements).sort(() => Math.random() - 0.5); // Перемешивание массива
```